models/migrations/migrations.go

```diff
@@ -464,8 +464,6 @@ func dropTableColumns(sess *xorm.Session, tableName string, columnNames ...strin
 			sess.Rollback()
 			return fmt.Errorf("Drop table `%s` columns %v: %v", tableName, columnNames, err)
 		}
```

Comment:
Also removed all `sess.Rollback() from the function?

Reply:
done